chsdi/__init__.py

```diff
@@ -49,49 +73,276 @@ def main(global_config, **settings):
+                     '/rest/services/{map}/MapServer/{layerId}/{featureId}/extendedHtmlPopup',
+                     request_method='GET')
+    add_cors_route(config,
+                   '/rest/services/{map}/MapServer/{layerId}/{featureId}/extendedHtmlPopup',
+                   'extendedHtmlPopup',
+                   methods=['GET'])
+    # Search using SphinxSearch
+    config.add_route('search',
+                     '/rest/services/{map}/SearchServer',
+                     request_method='GET')
+    add_cors_route(config,
+                   '/rest/services/{map}/SearchServer',
+                   'search',
+                   methods=['GET'])
+    # WMTS GetCapabilities document generation
+    config.add_route('wmtscapabilities',
+                     '/rest/services/{map}/1.0.0/WMTSCapabilities.xml',
+                     request_method='GET')
+    add_cors_route(config,
+                   '/rest/services/{map}/1.0.0/WMTSCapabilities.xml',
+                   'wmtscapabilities',
+                   methods=['GET'])
+    # Sends emails feedbacks
+    config.add_route('feedback', '/feedback', request_method='POST')
+    add_cors_route(config, '/feedback', 'feedback', methods=['POST'])
+    # Generates a qrcode
+    config.add_route('qrcodegenerator',
+                     '/qrcodegenerator',
+                     request_method='GET')
+    add_cors_route(config,
+                   '/qrcodegenerator',
+                   'qrcodegenerator',
+                   methods=['GET'])
+    # Generates a sitemap for SEO
+    config.add_route('sitemap', '/sitemap', request_method='GET')
+    add_cors_route(config,
+                   '/sitemap',
+                   'sitemap',
+                   methods=['GET'])
+    # Custom view for lufbilder individual images
+    config.add_route('luftbilder',
+                     '/luftbilder/viewer.html',
+                     request_method='GET')
+    add_cors_route(config,
+                   '/luftbilder/viewer.html',
+                   'luftbilder')
+    # Custom view for zeitreigen, pk50 and pk25 individual map sheets
+    config.add_route('historicalmaps',
+                     '/historicalmaps/viewer.html',
+                     request_method='GET')
+    add_cors_route(config,
+                   '/historicalmaps/viewer.html',
+                   'historicalmaps',
+                   methods=['GET'])
+    # Checks if the app is alive
+    config.add_route('checker',
+                     '/checker',
+                     request_method='GET')
+    add_cors_route(config,
+                   '/checker',
+                   'checker',
+                   methods=['GET'])
+    # Download a KML created in geoadmin
+    config.add_route('downloadkml',
+                     '/downloadkml',
+                     request_method='POST')
+    add_cors_route(config,
+                   '/downloadkml',
+                   'downloadkml',
+                   methods=['POST'])
```

Comment:
Why only POST?

Reply:
because it works only with a body and a post request.
See:
https://github.com/geoadmin/mf-chsdi3/blob/42c8b7199a8f59fb10253d1727e8fcb38b82d19f/chsdi/views/downloadkml.py#L54